microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -5,13 +5,14 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/27/2023
+ms.date: 08/14/2023
 ---
 # Extensions in the Microsoft Edge sidebar
 
 <!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/ -->
 
 This is a forthcoming feature; sidebar extensions are not yet in a stable release of Microsoft Edge.
```

Comment:
Is it ok to delete this sentence? --
This is a forthcoming feature; sidebar extensions are not yet in a stable release of Microsoft Edge.

@abhishek-shanthkumar, @Bhuvana-Priya

Rendered article section: first sentence of article:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-2677

Reply:
Hi @mikehoffms , now we do have sidebar in stable release for few versions but only for limited no. of developers.
What do you suggest can we add here instead of the current sentence?

Reply:
```
The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge.<!-- todo: delete sentence -->
```